CMakeLists.txt

```diff
@@ -250,17 +250,18 @@ message(STATUS "Using python bindings output path '${PYTHON_LIBRARY_OUTPUT_PATH}
 # environment variable $BOOST_ROOT is expected to point to non-standard locations
 message(STATUS "")
 message(STATUS ">>> Setting up Boost library.")
+set(Boost_USE_STATIC_LIBS      OFF)
+set(Boost_USE_MULTITHREADED    ON)
+set(Boost_USE_STATIC_RUNTIME   OFF)
+set(Boost_DETAILED_FAILURE_MSG ON)
```

Comment:
Do you know whether it has an effect to set `Boost_DETAILED_FAILURE_MSG` to `ON` if `REQUIRED` is used in `find_package` (see [here](https://cmake.org/cmake/help/v3.1/module/FindBoost.html))?
Anyway, it does not harm to set it!

Reply:
Unfortunately I also don't know what this will do exactly. Thought it might be useful. Let's see...